lib/generators/draft/resource/templates/controllers/controller.rb

```diff
@@ -1,13 +1,18 @@
 class <%= plural_table_name.camelize %>Controller < ApplicationController
   def index
-    @list_of_<%= plural_table_name.underscore %> = <%= class_name.singularize %>.all.order({ :created_at => :desc })
+    matching_<%= plural_table_name.underscore %> = <%= class_name.singularize %>.all
+
+    @list_of_<%= plural_table_name.underscore %> = matching_<%= plural_table_name.underscore %>.order({ :created_at => :desc })
 
     render({ :template => "<%= plural_table_name.underscore %>/index.html.erb" })
   end
 
   def show
     the_id = params.fetch("path_id")
-    @the_<%= singular_table_name.underscore %> = <%= class_name.singularize %>.where({ :id => the_id }).at(0)
+
+    matching_<%= class_name.singularize.downcase %> = <%= class_name.singularize %>.where({ :id => the_id })
```

Comment:
I think this should also be `plural_table_name`, for `matching_whatevers`?

Reply:
Ok I cleaned up the plural table names 

` the_id = params.fetch("path_id")

    matching_rsvp_tables = RsvpTable.where({ :id => the_id })

    @the_rsvp_table = matching_rsvp_tables.at(0)

    @the_rsvp_table.group_id = params.fetch("query_group_id")`